indico/modules/rb_new/client/js/common/config/reducers.js

```diff
@@ -32,11 +33,11 @@ export default combineReducers({
     data: (state = initialState, action) => {
         switch (action.type) {
             case configActions.CONFIG_RECEIVED: {
-                const config = action.data;
+                const {roomsSpriteToken, languages, tileServerURL} = camelizeKeys(action.data);
```

Comment:
are you sure it is gonna work? We return `{..., tileserver_url: <value>}` from the `RHConfig`. `camelizeKeys` will turn `tileserver_url` to `tileserverUrl`

Reply:
You are right. Thanks for pointing that out! 👍